bounded_counter/count_min_sketch.py

```diff
@@ -17,71 +17,148 @@ def hash_key(row, key):
 class CountMinSketch(object):
     """
     Data structure used to estimate frequencies of elements in massive data sets with fixed memory footprint.
+
+Example:
+        {
```

Comment:
What is this `{`?

Please use restructured text (.rst) for literal (code) formatting in docstrings.

In markdown files (.md), use Markdown (```python ...).